travis-scripts/jbossws-test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -ev
+
+
+# debug on non zero exit
+function finish {
+  if [[ $USE_WFLY_MASTER = "true" ]]; then
+     WFLY_TARGET=`pwd`"/wfly/wildfly/build/target/"
```

Comment:
You must use the full distribution that's built in dist/target directory, not the one in build/target, which uses maven references to avoid including actual module jars.